### ISSUE TITLE
Score

### ISSUE THREAD

**Issue by FieryPhoenix909** [fossasia/flappy-svg#223]
I downloaded the app on my phone and the clicked update before playing. The score doesn't work.
https://mail.google.com/mail/u/1/?ui=2&ik=77f0fe9668&view=att&th=159243ac32b9d2e3&attid=0.1&disp=safe&realattid=1554379222384902144-local0&z
https://mail.google.com/mail/u/1/?ui=2&ik=77f0fe9668&view=att&th=159243ac32b9d2e3&attid=0.2&disp=safe&realattid=1554379222384902144-local1&z
https://mail.google.com/mail/u/1/?ui=2&ik=77f0fe9668&view=att&th=159243ac32b9d2e3&attid=0.3&disp=safe&realattid=1554379222384902144-local2&z
https://mail.google.com/mail/u/1/?ui=2&ik=77f0fe9668&view=att&th=159243ac32b9d2e3&attid=0.4&disp=safe&realattid=1554379222384902144-local3&zwwww

**Comment by niccokunzmann** [comment]
Does it also not work on the website?
The score updates, when you cross a obstacle. Did you cross obstacles?

**Comment by Abhi2424shek** [comment]
@niccokunzmann it's working on the website. We need to update the script on the apk to contain the new version.

**Comment by niccokunzmann** [comment]
It depends on the app. The app used here has an update functionality.

**Comment by Abhi2424shek** [comment]
@FieryPhoenix909 are you using the latest version...

**Comment by FieryPhoenix909** [comment]
Yes, I downloaded it from the google play store and made sure to click the update button in the app before playing if that's what you mean.

@niccokunzmann The website works great and I am passing obstacles on my phone

**Comment by FieryPhoenix909** [comment]
I also wanted to point out that this is what the help screen looks on my phone. Not sure if my phones the problem but just so you know.
![screenshot_2016-12-22-11-10-37](https://cloud.githubusercontent.com/assets/23224432/21432291/c985b6ac-c838-11e6-91dd-963f18f24ea1.png)

**Comment by Abhi2424shek** [comment]
@FieryPhoenix909 any updates

**Comment by Marauderer97** [comment]
@FieryPhoenix909 Can you try loading the help screen again, you might have used an old link, and the nav bar links in the main menu have been recently updated

**Comment by Abhi2424shek** [comment]
@FieryPhoenix909 updates?

**Comment by FieryPhoenix909** [comment]
The help screen is still like that and the score still doesn't work. I don't think i'm doing anything wrong in updating the app and playing and stuff. Srry I took so long to reply. got caught up with midterms and school etc. XD

**Comment by Abhi2424shek** [comment]
We would fix the credits page then
